sass/utilites/__mixin.scss

```diff
@@ -0,0 +1,56 @@
+@mixin clear-fix {
+  &::after {
+    content: '';
+    display: table;
+    clear: both;
+  }
+}
+
+@mixin vh-align {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+// don't forget pass the ($url) as a string
+@mixin background-image-effect($url) {
+  & {
+    background-image:
+      linear-gradient(to top, rgba(#000, 0.8) 15%, transparent),
+      url($url);
+    background-size: cover;
+    background-position: 50%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+  }
+}
+
+// Responsive Media Query
+// tablet = 768 px
+// laptop = 1024
+@mixin respond($breakpoint) {
+  @if $breakpoint == 'tablet' {
+    @media (min-width: 768px) {
+      @content;
+    }
+  }
+
+  @else if $breakpoint == 'laptop' {
+    @media (min-width: 1440px) {
```

Comment:
Fix this media query I see that the tablet version is shown on the desktop width, you need to change 1440px to 1024px:
![image](https://user-images.githubusercontent.com/34653764/87204008-b9eb3200-c2fb-11ea-8ff4-15132b95f4f5.png)

Reply:
Done